### PR TITLE
[SKOOP-143] Success messages for the user profile page

### PR DESCRIPTION
**Pull request by svetivanova**
A service for openning success/warning/error popup notifications was added. 
Popup notification is displayed after each successfully executed create/update/delete operation for each section of the "My Profile" page.
@georgwittberger, @LebedkoDmitry could you please take a look?

### CODE REVIEW

File: src/app/shared/popup-notification.service.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  showWarningMessage(message: string) {
+    this.openSnackBar(message, 'warning-notification');
+  }
+
+  showErrorMessage(message: string) {
+    this.openSnackBar(message, 'error-notification');
+  }
+
+  private openSnackBar(message: string, className: string) {
```
Comment (LebedkoDmitry) [inline src/app/shared/popup-notification.service.ts]: Wouldn't it be better to restrict `className` with the aid of some enumeration like `className: POPUP_NOTIFICATION_CLASSNAME` that maps to an actual classname?

**Comment by LebedkoDmitry** [comment]
I think I got Georg's point. Perhaps it would be better to have a panel on the top of User Profile page instead of Snack Bar to display messages on successful fullfilment of the operation. See the image below.

Georg, please correct me if I am wrong.

image

**Comment by georgwittberger** [comment]
> ... Perhaps it would be better to have a panel on the top of User Profile page instead of Snack Bar to display messages on successful fullfilment of the operation. ...

This is what I had in mind during the last review meeting - a colored line of text at the top of the expandable panel (inside each panel). Anyway, I would keep the snack bars for now. If someone complains about the definitely more beautiful snack bar solution then we can still change it.

File: src/app/shared/popup-notification.service.ts (hunk 1 above)
Comment (svetivanova) [reply src/app/shared/popup-notification.service.ts]: Good point. Thank you!

**Pull request merged**